internal/driver/gl/canvas_test.go

```diff
@@ -77,3 +76,86 @@ func Test_glCanvas_SetContent(t *testing.T) {
 		})
 	}
 }
+
+func Test_glCanvas_GrowingMinSize(t *testing.T) {
```

Comment:
Is it possible to shrink this test down or split it into multiple?
It's hard to read as is and not all items are checked in the assert. This is an open question and if this is the minimum required that's fine.

Reply:
Yeah, I could split it up.